### PR TITLE
tslint rule interface name: ivisualization-store-data

### PR DESCRIPTION
**Pull request by smoralesd**
#### Description of changes

Rename file and interfaces inside ivisualization-store-data file.

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000 **N/A**
- [ ] Added relevant unit test for your changes. (`npm run test`) **N/A**
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage` **N/A**
- [x] Ran precheckin (`npm run precheckin`)
- [ ] Added screenshots/GIFs for UI changes. **N/A**

### CODE REVIEW

File: src/assessments/assessment-builder.tsx [hunk 1]
```diff
@@ -87,21 +87,21 @@ export class AssessmentBuilder {
...
+    private static enableTest(scanData: ScanData, payload: AssessmentToggleActionPayload): void {
+        const scanAssessmentData = scanData as AssessmentScanData;
         scanAssessmentData.enabled = true;
         scanAssessmentData.stepStatus[payload.requirement] = true;
     }
 
-    private static disableTest(scanData: IScanData, requirement: string): void {
-        const scanAssessmentData = scanData as IAssessmentScanData;
-        scanAssessmentData.stepStatus[requirement] = false;
+    private static disableTest(scanData: ScanData, step: string): void {
```
Comment (dbjorge) [inline src/assessments/assessment-builder.tsx]: This looks like it's reverting an earlier change to rename step to requirement. Bad merge?

Comment (smoralesd) [reply src/assessments/assessment-builder.tsx]: Bad merge indeed. Thanks for noticing.

Comment (smoralesd) [reply src/assessments/assessment-builder.tsx]: @dbjorge Updated

File: src/assessments/assessment-builder.tsx [hunk 2]
```diff
@@ -87,21 +87,21 @@ export class AssessmentBuilder {
...
         scanAssessmentData.stepStatus[requirement] = false;
         scanAssessmentData.enabled = Object.keys(scanAssessmentData.stepStatus).some(key => scanAssessmentData.stepStatus[key] === true);
     }
 
-    private static getTestStatus(scanData: IScanData, requirement: string): boolean {
-        const scanAssessmentData = scanData as IAssessmentScanData;
-        return requirement in scanAssessmentData.stepStatus && scanAssessmentData.stepStatus[requirement];
+    private static getTestStatus(scanData: ScanData, step: string): boolean {
+        const scanAssessmentData = scanData as AssessmentScanData;
+        return step in scanAssessmentData.stepStatus && scanAssessmentData.stepStatus[step];
```
Comment (waabid) [inline src/assessments/assessment-builder.tsx]: Should be requirement here

File: src/assessments/assessment-builder.tsx [hunk 3]
```diff
@@ -87,21 +87,21 @@ export class AssessmentBuilder {
...
+    private static disableTest(scanData: ScanData, requirement: string): void {
+        const scanAssessmentData = scanData as AssessmentScanData;
         scanAssessmentData.stepStatus[requirement] = false;
         scanAssessmentData.enabled = Object.keys(scanAssessmentData.stepStatus).some(key => scanAssessmentData.stepStatus[key] === true);
     }
 
-    private static getTestStatus(scanData: IScanData, requirement: string): boolean {
-        const scanAssessmentData = scanData as IAssessmentScanData;
-        return requirement in scanAssessmentData.stepStatus && scanAssessmentData.stepStatus[requirement];
+    private static getTestStatus(scanData: ScanData, step: string): boolean {
```
Comment (waabid) [inline src/assessments/assessment-builder.tsx]: requirement*

**Pull request closed without merging**